runtime/src/attach/attach_manager/frida_attach_manager.cpp

```diff
@@ -368,21 +373,21 @@ void frida_internal_attach_entry::iterate_uretprobe_callbacks(
 
 using namespace bpftime;
 
-extern "C" uint64_t __bpftime_frida_attach_manager__replace_handler()
-{
-	GumInvocationContext *ctx;
-	pt_regs regs;
-
-	ctx = gum_interceptor_get_current_invocation();
-	convert_gum_cpu_context_to_pt_regs(*ctx->cpu_context, regs);
-	frida_internal_attach_entry *hook_entry =
-		(frida_internal_attach_entry *)
-			gum_invocation_context_get_replacement_data(ctx);
-	auto &cb = hook_entry->get_replace_callback();
-	uint64_t ret = cb(regs);
-	gum_invocation_context_replace_return_value(ctx, (gpointer)ret);
-	return ret;
-}
+// extern "C" uint64_t __bpftime_frida_attach_manager__replace_handler()
```

Comment:
Remove them.

Comment out code is unnecessary and will bring more maintaining problems, so please don't do that.

Reply:
Fixed now